sebs/gcp/storage.py

```
@@ -30,16 +67,42 @@ def replace_existing(self, val: bool):
+        """Correct bucket name to meet GCP naming requirements.
+        
+        Args:
+            name: Original bucket name
+            
+        Returns:
+            Corrected bucket name (no changes needed for GCP)
+        """
         return name
 
-    def _create_bucket(self, name, buckets: List[str] = [], randomize_name: bool = False):
+    def _create_bucket(self, name: str, buckets: List[str] = [], randomize_name: bool = False) -> str:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)